src/scenes/Users/List/UserList.tsx

```diff
@@ -23,23 +23,9 @@ import { UserSortOptions } from './UserSortOptions';
 
 const TabList = ActualTabList as typeof __Tabs;
 
-const useStyles = makeStyles()(({ spacing, breakpoints }) => ({
-  options: {
-    margin: spacing(3, 0),
-  },
-  items: {
-    maxWidth: breakpoints.values.sm,
-  },
-  tabPanel: {
-    overflowY: 'auto',
-    // allow card shadow to bleed over instead of cutting it off
-    padding: spacing(0, 0, 0, 2),
-    margin: spacing(0, 0, 0, -2),
-  },
-  total: {
-    marginTop: spacing(2),
-  },
-}));
+const itemsSx = (theme: Theme) => ({
```

Comment:
I wonder if it would be better to name this like it's purpose, "max width", instead of where it's used. Or maybe "itemWidth"?